### ISSUE TITLE
dapr dashboard -k fails on MacOS

### ISSUE THREAD

**Issue by artursouza** [dapr/cli#1091]
## Expected Behavior

`dapr dashboard -k` should open the browser at the dapr dashboard endpoint using CLI 1.9.0-rc.2

## Actual Behavior

![image](https://user-images.githubusercontent.com/130954/194997440-6718667e-54f6-4507-8cf9-ac69df620170.png)

## Steps to Reproduce the Problem

`dapr dashboard -k` on MacOS

## Release Note

RELEASE NOTE:

**Comment by artursouza** [comment]
I confirmed the same did not happen with CLI 1.8.0-rc.1 (tested with RC because it was easy for me via Homebrew).

**Comment by mukundansundar** [comment]
The latest build changed 
this
```go
var webURL string = fmt.Sprintf("http://%s:%d", dashboardHost, dashboardLocalPort) //nolint:nosprintfhostport
```
to this
```
var webURL string = net.JoinHostPort(dashboardHost, fmt.Sprint(portForward.LocalPort))
```

without the scheme ... That breaks the browser open URL line.

**Comment by mukundansundar** [comment]
This is a regression and needs to be fixed with another RC ...

**Comment by mukundansundar** [comment]
closed by #1092
